daml-lf/archive/BUILD.bazel

```diff
@@ -46,9 +46,15 @@ load("@rules_pkg//:pkg.bzl", "pkg_tar")
         ),
     ]
     for version in PROTO_LF_VERSIONS
-    for short_version in ["dev" if version == "1.dev" else version]
+    for short_version in ["dev" if version in [
+        "1.dev",
+        "2.dev",
+    ] else version]
```

Comment:
I'm not sure if it's a problem that `{1,2}.dev` have the same value of `maven_artifact_prefix`,

Reply:
Yes, it isn't clean, I just tried to get something building but I want to make that cleaner.